VideoLocker/res/values/strings.xml

```diff
@@ -83,6 +83,8 @@
     <string name="dropdown_list_prefix">Dropdown list.</string>
 
     <!-- Navigation Drawer Labels -->
+    <!-- The accessibility description for the navigation header -->
+    <string name="navigation_header">Signed in with username {username} and email {email}</string>
```

Comment:
Google app only announces the email, but has name and email visually. We have a user's name (not the username) and email. Should we stick to just the email?